wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -2891,6 +3091,46 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 			<wsdl:input message="tds:GetWsdlUrlRequest"/>
 			<wsdl:output message="tds:GetWsdlUrlResponse"/>
 		</wsdl:operation>
+		<wsdl:operation name="GetPasswordComplexityOptions">
+			<wsdl:documentation>This method allows to retrieve all the available parameters and their valid ranges for the password complexity configuration.</wsdl:documentation>
```

Comment:
I feel this should read ;

This method allows retrieval of all the available parameters and their valid ...